screens/Reader.js

```diff
@@ -94,18 +95,19 @@ class Reader extends React.Component {
     if (startBaniList.length > 0) {
       for (const bani of startBaniList) {
         if (bani.id == this.currentBani.id) {
-          if (bani.progress > 0 && bani.progress < 1) {
             progress = bani.progress;
-          }
         }
       }
     }
+    if(Number(progress)==1 || Number(progress)>"1"){
+      progress=0
+    }
     this.currentBani.progress = progress;
   }
 
-  handleBackPress(data) {
-    this.webView.postMessage(JSON.stringify({ Back: true }));
-    this.props.navigation.goBack();
+  handleBackPress=()=> {
+   this.webView.postMessage(JSON.stringify({ Back: true }));
+    setTimeout(()=>{this.props.navigation.goBack();},100);
```

Comment:
Veerji indentation is still an issue in this PR at multiple places. I am opening an issue to configure eslint with husky and after that, all the files have to be formatted correctly. I'm letting the indentation slide in this PR.

Reply:
Fixed Veer ji